dali/test/python/test_external_source_dali.py

```diff
@@ -412,6 +413,8 @@ def test_layout_changing():
     src_pipe.build()
     src_pipe.feed_input("input", [np.zeros((1))], layout="W")
     src_pipe.feed_input("input", [np.zeros((1))], layout="H")
+    src_pipe.run()
+    src_pipe.run()
```

Comment:
Since `ValidateInputs` are now in `SetupImpl`, not `SetDataSource` (so a step later), this test had to be updated.